docs/source/PyMOLplugin.rst

```diff
@@ -0,0 +1,233 @@
+PyMOL plugin
+============
+
+This plugin provides tools for identification, classification and analysis of
+conserved water molecules and their networks from molecular dynamics
+simulation trajectories.
+
+The main interface consists of the following sections:
+
+Input Options
+-------------
+
+Input trajectories **MUST** be aligned and centered before analysis. The
+plugin will not perform any alignment or centering. If the system is
+centered on the protein, it usually works well. However, in instances
+where the area of interest is close to the edge of the simulation box,
+system should be centered on the position of the center of mass of the area of
+interest.
+
+**Selection Input Type**: Choose the type of input you want to provide. Options include:
+
+.. tabs:: 
+
+  .. tab:: **PyMOL Selection**
+
+    A PyMOL selection is first exported to a PDB file and waters are
+    extracted using water selection panel using MDAnalysis.
+
+    **PyMOL Selection**: Enter the selection string which contains
+    waters (can contain other elements as well) in your PyMOL window. We
+    recommend selection contains system as well as waters of interest.
```

Comment:
Enter the selection string in your PyMOL window, ensuring it contains water molecules (though it may contain other elements as well). It is recommended that the selection includes both the system and the water molecules of interest.